doc/source/rllib/index.rst

```diff
@@ -66,103 +66,115 @@ for a few training iterations, then perform a single evaluation loop
+    .. container:: buttons-float-left
 
-.. .. container:: clear-both
+        .. image:: ../images/rllib/sigils/rllib-sigil-multi-agent.svg
+            :width: 100
+            :target: https://github.com/ray-project/ray/blob/master/rllib/examples/multi_agent_independent_learning.py
 
-..    .. container:: buttons-float-left
+    .. container::
 
-..        .. image:: ../images/rllib/sigils/external-simulators.svg
-..            :width: 120
+        | **Multi-agent RL** (MARL): Convert your (custom) ``gym.Env``s into a multi-agent ones
+          via a few simple steps and start training your agents in any of the following fashions:
+        | 1) Cooperative with `shared <https://github.com/ray-project/ray/blob/master/rllib/examples/centralized_critic.py>`_ or
+          `separate <https://github.com/ray-project/ray/blob/master/rllib/examples/two_step_game.py>`_
+          policies and/or value functions.
+        | 2) Adversarial scenarios using `self-play <https://github.com/ray-project/ray/blob/master/rllib/examples/self_play_with_open_spiel.py>`_
+          and `league-based training <https://github.com/ray-project/ray/blob/master/rllib/examples/self_play_league_based_with_open_spiel.py>`_.
+        | 3) `Independent learning <https://github.com/ray-project/ray/blob/master/rllib/examples/multi_agent_independent_learning.py>`_
+          of neutral/co-existing agents.
 
-.. container::
 
-    **External simulators** connecting to RLlib from the outside (e.g.
-    via http(s)): Don't have your simulation running as a gym.Env in python?
-    No problem, RLlib supports an external environment API and comes with a pluggable,
-    off-the-shelve client/server setup that allows you to run 100s of independent
-    simulators on the "outside" (e.g. a Windows cloud) connecting to a central RLlib
-    Policy-Server that learns and serves actions. Alternatively, actions can be computed
-    on the client side to save on network traffic.
+.. container:: clear-both
 
-.. .. container:: clear-both
+    .. container:: buttons-float-left
 
-..    .. container:: buttons-float-left
+        .. image:: ../images/rllib/sigils/rllib-sigil-external-simulators.svg
+            :width: 100
+            :target: https://github.com/ray-project/ray/tree/master/rllib/examples/serving
 
-..        .. image:: ../images/rllib/sigils/offline-rl.svg
-..            :width: 120
+    .. container::
 
-.. container::
+        **External simulators**: Don't have your simulation running as a gym.Env in python?
+        No problem! RLlib supports an external environment API and comes with a pluggable,
+        off-the-shelve client/server setup that allows you to run 100s of independent
+        simulators on the "outside" (e.g. a Windows cloud) connecting to a central RLlib
+        Policy-Server that learns and serves actions. Alternatively, actions can be computed
+        on the client side to save on network traffic.
```

Comment:
is there an example for this actually? or code pointer?

Reply:
Yeah, I linked the directory with the client/server examples (Unity3D and CartPole) in the sigil.
I can add a link to the text as well.

Reply:
done. Pointed directly to the "client" and "server" separate example scripts in the test. The sigil links into the entire "examples/serving" directory.